pkg/dashboards/types.go

```diff
@@ -265,14 +265,6 @@ type DashboardWidgetNRQLQueryInput struct {
 	Query nrdb.NRQL `json:"query"`
 }
 
-// DashboardWidgetQueryInput -
```

Comment:
Where did this go?

Reply:
Honestly not sure 🤷  It's not referenced anywhere as far as I can tell.